cordform-common/src/main/java/net/corda/cordform/CordformNode.java

```diff
@@ -43,6 +43,13 @@ public String getName() {
 
     public int getRpcPort() { return rpcPort; }
 
+    private Integer sshdPort;
```

Comment:
The "getter" is returning `Integer` but the setting is receiving `int`. Does Java/Kotlin still recognise `sshdPort` as a property, despite their *non-identical* types? (I think I might have been bitten by something like this in the past.)

Or put another way, why not pass `Integer` into the setter too? Might people want to _unset_ an optional field?